powershell/ql/lib/semmle/code/powershell/security/CommandInjectionCustomizations.qll

```diff
@@ -39,13 +42,16 @@ module CommandInjection {
     SystemCommandExecutionSink() {
       // An argument to a call
       exists(DataFlow::CallNode call |
-        call.getName() = "Invoke-Expression" and
+        call.getName() = ["Invoke-Expression", "iex"] and
```

Comment:
We should probably create a subclass of `DataFlow::CallNode` to represent calls to `Invoke-Expression` since there are apparently many names for this (as you've discovered!)

That would also allow us to centralize the `getName` call to a single place (and make it all lowercase so that we also match stuff like `Iex` since I'm sure that also resolves to `Invoke-Expression`)